Gemfile

```diff
@@ -5,7 +5,8 @@ require File.join(Bundler::Plugin.index.load_paths("bundler-inject")[0], "bundle
 
 gem 'json-schema',       '~> 2.8'
 gem 'manageiq-loggers',  '~> 0.1'
-gem 'openapi_parser',    '~> 0.3.0'
+# In 0.3.1 additionalProperties of objects default to true, we have tu put everywhere additionalProperites : false
```

Comment:
`s/tu/to/`

Reply:
hehe, but my IDE is marking `tu` as valid, so it must be right :-D

Reply:
:laughing: